README.md

```diff
@@ -1,8 +1,6 @@
 xgboost-predictor-java
 ======================
-
-[![Build Status](https://travis-ci.org/komiya-atsushi/xgboost-predictor-java.svg?branch=master)](https://travis-ci.org/komiya-atsushi/xgboost-predictor-java)
-[![Download](https://api.bintray.com/packages/komiya-atsushi/maven/xgboost-predictor/images/download.svg) ](https://bintray.com/komiya-atsushi/maven/xgboost-predictor/_latestVersion)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/ai.h2o/xgboost-parser/badge.svg)](https://maven-badges.herokuapp.com/maven-central/ai.h2o/xgboost-parser)
```

Comment:
almost good, just poiting to xgboost-parser instaed of xgboost-predictor :)

Reply:
omg a stupid mistake, thanks for double check!